include/eve/module/core/regular/bit_ternary.hpp

```diff
@@ -325,7 +332,7 @@ namespace eve
       if constexpr(K == 0xc7) return bit_or(bit_notand(c, b), bit_xor(a, bit_not(b)));
       if constexpr(K == 0xc8) return bit_and(b, bit_or(a, c));
       if constexpr(K == 0xc9) return bit_not(bit_xor(b, bit_or(a, c)));
-      if constexpr(K == 0xca) return bit_or(bit_and(a, b), bit_notand(a, c));
+      if constexpr(K == 0xca) return bit_select(a, b, c); //bit_or(bit_and(a, b), bit_notand(a, c));
```

Comment:
clang format please. The comments are not formatted.

Reply:
I want there one line by constexpr here. If you prefer I can suppress the comments.
However, what is the comment format you want here ?